iOS/Flows/Conversation/Conversation List/ConversationListViewController.swift

```diff
@@ -227,13 +232,19 @@ class ConversationListViewController: InputHandlerViewContoller, ConversationLis
         // Determine if this is a reply message or regular message. If it's a reply, select the parent
         // message so we can open the thread experience.
         if let parentMessageId = message.parentMessageId {
-            await messagesCell.scrollToMessage(with: parentMessageId, animateScroll: true, animateSelection: true)
+            await messagesCell.scrollToMessage(with: parentMessageId, animateScroll: animateScroll, animateSelection: animateSelection)
 
             if let messageCell = messagesCell.getFrontmostCell() {
                 self.messageContentDelegate?.messageContent(messageCell.content, didTapMessage: (cid, messageId))
             }
+        } else if viewReplies {
+            await messagesCell.scrollToMessage(with: messageId, animateScroll: animateScroll, animateSelection: animateSelection)
```

Comment:
Add this case because sometimes we want to just go straight to replies, even though there might not be any.